stratum/hal/config/x86-64-stordis-bf2556x-1t-r0/phal_config.pb.txt

```diff
@@ -1,283 +1,283 @@
 cards {
-  id: 1
+  slot: 1
   ports {
-    id: 1
+    port: 1
     physical_port_type: PHYSICAL_PORT_TYPE_QSFP_CAGE
```

Comment:
Are these really QSFP cages? I thought the first 48 ports are SFP+... will check tomorrow....

Reply:
I don't know, but this is not part of this PR. Default configs were merged in #58. Make a new issue if they spot any errors.

Reply:
@stolsma You are right, I think I copied the wrong config. I can create another PR to fix that

Thanks